src/helpers/cameraHelper.ts

```diff
@@ -1,4 +1,4 @@
-import { Camera } from 'expo-camera';
+import { Camera } from 'expo-camera/legacy';
```

Comment:
The `expo-camera` module in the Expo framework allows developers to use the camera functionalities within their React Native applications. Over time, the module has evolved, leading to different versions or approaches to handling camera features. Understanding the differences between `expo-camera/legacy` and `expo-camera/next` involves recognizing their respective design goals, feature sets, and improvements.

### `expo-camera/legacy`
1. **Older API**: This version represents the initial implementation of the camera module. It includes the fundamental features required to access and use the device camera within an Expo app.
2. **Stability**: As a legacy version, it is generally more stable and has been tested across various projects over time.
3. **Limited Features**: While stable, it may not support newer features or optimizations introduced in later versions.
4. **Maintenance**: This version might not receive new features or significant updates, focusing mainly on critical bug fixes and maintaining existing functionality.

### `expo-camera` / `expo-camera/next`
1. **Modern API**: This represents the latest implementation of the camera module, leveraging modern practices and potentially introducing new capabilities.
2. **Enhanced Features**: Includes improvements and new features that were not present in the legacy version. These could be related to performance, ease of use, additional functionalities, and better integration with the latest React Native and Expo SDK versions.
3. **Optimization**: Likely to include performance enhancements and optimizations, making it more efficient compared to the legacy version.
4. **Active Development**: Being the newer version, it is under active development and is likely to receive regular updates, new features, and improvements based on user feedback and evolving standards.

### Key Differences
- **API Design**: `expo-camera/next` might introduce changes or simplifications in the API to make it more intuitive or aligned with modern React Native practices.
- **Performance**: The next version is likely optimized for better performance, leveraging newer technologies and methodologies.
- **Feature Set**: New features and enhancements that are not present in the legacy version.
- **Support and Updates**: The next version receives ongoing support and updates, while the legacy version is maintained only for stability and critical fixes.

When choosing between the two, developers should consider the stability and feature needs of their project. For projects that require the latest features and optimizations, `expo-camera/next` would be the better choice. However, for projects where stability is paramount and the existing features of the legacy version are sufficient, `expo-camera/legacy` might be more appropriate.

Reply:
There is no current need for this upgrade, it becomes mandatory with SDK 52 therefore I'd make it with the Expo SDK 52 upgrade since its more efficent to keep up the development und push 4.0.0 with the current expo camera version.